src/libraries/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.LoadICU.iOS.cs

```diff
@@ -7,8 +7,10 @@ internal static partial class GlobalizationMode
     {
         private static int LoadICU()
         {
-            object? datPath = AppContext.GetData("ICU_DAT_FILE_PATH");
-            return (datPath != null) ? Interop.Globalization.LoadICUData(datPath!.ToString()!) : Interop.Globalization.LoadICU();
+            // NativeAOT doesn't set ICU_DAT_FILE_PATH so we fallback to icudt.dat in the app bundle root in that case
```

Comment:
It would be less ceremony if we allowed passing null to `LoadICUData` and let `LoadICUData` to replace it with the default.

Reply:
ok

Reply:
Fixed